include/settings.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PIPE_NETWORK_SETTING_H_
+#define PIPE_NETWORK_SETTING_H_
+
+namespace pipenetwork {
+//! define gravitational acceleration in x, y, z direction
+const Eigen::Vector3d g(0.00, 0.00, 9.81);
```

Comment:
Could you please show me where you found this way of initialisation? using () for initilisation?

Reply:
https://eigen.tuxfamily.org/dox/group__TutorialMatrixArithmetic.html